packages/generator-rest/src/request-builder/api-request-builder-class.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  ClassDeclarationStructure,
+  MethodDeclarationStructure,
+  OptionalKind,
+  ParameterDeclarationStructure,
+  StructureKind
+} from 'ts-morph';
+import { toPascalCase, toPropertyFormat } from '@sap-cloud-sdk/core';
+import { OpenApiPath, OpenApiServiceMetadata } from '../open-api-types';
+
+export function apiRequestBuilderClass(
```

Comment:
add some doc, that this methods creates the wrapper class right?

Reply:
yes, all the files in this folder are ts-morph related files for generating our wrapper classes.

Reply:
Added more comments.